src/components/tooltip/Tooltip.js

```diff
@@ -243,17 +243,17 @@ class Tooltip extends React.Component<Props, State> {
 
         const bodyEl = bodyElement instanceof HTMLElement ? bodyElement : document.body;
 
-        const classes = classNames('tooltip', className, {
+        const classes = classNames('bdl-Tooltip', className, {
             'is-callout': theme === CALLOUT_THEME,
```

Comment:
It's a little awkward that some of the modifiers are transformed and some aren't within the same component.